src/components/SlateEditor/plugins/link/Link.tsx

```diff
@@ -115,7 +111,7 @@ const Link = (props: Props) => {
       let checkbox;
       if (element.type === 'content-link') {
         const contentType = element['content-type'] || 'article';
-        href = `${await fetchResourcePath(element, language, contentType)}`;
+        href = `${fetchResourcePath(element, language, contentType)}`;
```

Comment:
Denne vil vise url til learning-resource sjølv om det er snakk om et emne. Kanskje du kan få til at det står topic-article heller for disse?